tests/python/driver/tvmc/test_compiler.py

```diff
@@ -29,6 +30,15 @@
 from tvm.driver import tvmc
 
 
+def vitis_ai_available():
```

Comment:
Minor comment: this function could be useful on your main module, near `partition_for_vitis_ai`.

Reply:
Yes, I wanted to do that but those files import the `pyxir` package, which might already fail if the package is not installed. So, unless we put the function before the imports, importing this function would cause failure. We can put the function in a separate file (e.g. `vitis_ai_tools`) but that felt a bit overkill currently as we are only using it in one place. If we start needing it in multiple places, then I think we should definitely move it.